app/src/main/res/values/strings.xml

```diff
@@ -102,6 +102,16 @@
     <string name="developer_options_msaa_2">2x</string>
     <!-- This string is used to label the MSAA radio button that enables four times MSAA in immersive mode. -->
     <string name="developer_options_msaa_4">4x</string>
+    <!-- This string is used to label radio buttons for setting fixed-foveated-rendering -->
```

Comment:
can we add a note that explains in a little bit more detail what this means. I assume this string will be difficult to translate for most.